mvs_eland_tool/server_deploy.py

```diff
@@ -0,0 +1,100 @@
+"""
+This is the main file of the tool "Multi-vector simulation tool".
+
+Tool structure:
+
+(parent)    mvs_eland_tool.py
+(child)     --A0_initialization.py
+
```

Comment:
```suggestion
(child)     --A0_initialization.py
(child sub)    -- A1_csv_to_json.py

```

Reply:
The server deploy will never use the csv_to_json feature